commands.go

```diff
@@ -266,7 +266,7 @@ func adilioMessage(s *discordgo.Session, m *discordgo.MessageCreate) {
 	span.AddField("command", "AdilioLol")
 
 	if strings.Contains(strings.ToLower(m.Message.Content), "lol") {
-		sendResponse(ctx, s, m.ChannelID, "<:adilolol:769432234880925727> <:adilol:769263097772245032>")
+		sendResponse(ctx, s, m.ChannelID, "<:adilio:769432234880925727> <:adilol:769263097772245032>")

```

Comment:
```suggestion
		sendResponse(ctx, s, m.ChannelID, "<:adilio:788826086628261889> <:adilol:769263097772245032>")

```
Need to use the ID of the emote.